ipaserver/plugins/rabase.py

```diff
@@ -44,8 +44,8 @@ def __init__(self, api):
             self.sec_dir = api.env.dot_ipa + os.sep + 'alias'
             self.pwd_file = self.sec_dir + os.sep + '.pwd'
         else:
-            self.sec_dir = paths.IPA_RADB_DIR
-            self.pwd_file = os.path.join(paths.IPA_RADB_DIR, 'pwdfile.txt')
+            self.sec_dir = paths.HTTPD_ALIAS_DIR
+            self.pwd_file = os.path.join(paths.HTTPD_ALIAS_DIR, 'pwdfile.txt')
```

Comment:
This does not make sense anymore, the same as in `RestClient.__init__()` should be done here.